### DISCUSSION TITLE
p/c46c93fd63774dc7952eb72a52b83650/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#21] [Announcements]
# p/c46c93fd63774dc7952eb72a52b83650/

想做的事情很多，但时间总是太少。吾辈这周的主要精力仍然没有放在日语上面，更多的还是处理工作上的事情，web chat 的开发进展有点慢，下周调整状态做的更好。 开发这周公司终于开始认真打算尝试 chatgpt 如何集成到项目中了，虽然相对于社区的各路神仙而言这已经是起步太慢了，但似乎相比于更多不愿醒来的人而言已经算是一个进步了？吾辈这周日也终于将用户系统简单实现了，虽然还未想要的集成会话和消息同步

https://blog.rxliuli.com/p/c46c93fd63774dc7952eb72a52b83650/

**Comment by miaopass-future** [comment]
打算做chatGPT 收费转发类型应用么

**Reply by rxliuli** [reply]
或许以后会考虑吧，至少目前没有打算，可能如果用的太多会实现限速 + 支持自定义 OPENAI_API_KEY。不过目前还在探索 gpt 的能力限制，感觉 langchain 会是一个非常有趣的尝试。

**Reply by rxliuli** [reply]
用户系统实现主要是为了支持同步会话和消息的功能，吾辈正在 coding 中。。。

**Reply by miaopass-future** [reply]
啥场景会用到比较长篇的会话和消息？

**Reply by rxliuli** [reply]
其实还是挺多的，例如需要多次询问特定领域的问题时通常如果容易找到就不会新开会话。例如
![image](https://user-images.githubusercontent.com/24560368/229690750-c1fb11f5-d49a-4a81-aca5-56fcc4a7ade8.png)

而且目前 openai 的计费机制吾辈还没真正搞清楚，某些消息 tokens 使用超过 152k，这是后面必须解决的问题之一。
ref: https://community.openai.com/t/what-is-n-context-tokens-total-why-is-it-so-large/127420

**Reply by miaopass-future** [reply]
新开会话一般也都是新的问题了吧。AI目前的能力感觉没那么强，对于上下文来说，据我使用，越长感觉出错的概率更大。想要得到更精准的回答，需要更少的问题段。最近听说封号特别严重。。。。。

**Reply by rxliuli** [reply]
> 新开会话一般也都是新的问题了吧。

吾辈原本的预期也是这个，但看使用数据不是

> AI 目前的能力感觉没那么强，对于上下文来说，据我使用，越长感觉出错的概率更大。想要得到更精准的回答，需要更少的问题段。

有局限性但还行，gpt4 的改善比较明显，不过目前还在 waitlist

> 最近听说封号特别严重。。。。。

最近封号严重倒是真的，吾辈同事都开始说起这件事了。
